src/components/TempTodo/TempTodo.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { useTodos } from '../../Store';
+
+export const TempTodo: React.FC = () => {
+  const { tempTodo, handleChangeStatus } = useTodos();
+
+  return (
+    <>
+      {tempTodo && (
```

Comment:
better move the conditional rendering outside the component and don't render it at all if there is no `tempTodo`